app-frontend/src/app/pages/lab/edit/edit.controller.js

```diff
@@ -1,27 +1,44 @@
 export default class LabEditController {
-    constructor($scope, $element) {
+    constructor($scope, $element, $uibModal) {
         'ngInject';
         this.$scope = $scope;
         this.$element = $element;
-        this.$parent = $scope.$parent.$ctrl;
-        this.isShowingParams = false;
+        this.$uibModal = $uibModal;
     }
 
     $onInit() {
-        if (this.$parent.toolRequest) {
-            this.$parent.toolRequest.then(t => {
-                this.cellLabel = t.title;
-            });
-        }
+        this.inputs = [false, false];
```

Comment:
This feels a bit icky but it's not really worth putting any additional effort into this since it's just demo code for the most part.